### PR TITLE
Modify class not be initialized during import

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

I receive the following errors simply using `thoth.messaging import MessageBase` when I run tests:

```
=============================================================================================================================== ERRORS ================================================================================================================================
____________________________________________________________________________________________________ ERROR collecting tests/test_investigate_unresolved_package.py ____________________________________________________________________________________________________
tests/test_investigate_unresolved_package.py:25: in <module>
    from thoth.investigator.investigate_unresolved_package import investigate_unresolved_package
thoth/investigator/__init__.py:24: in <module>
    from thoth.messaging import __version__ as __messaging__version__
../../../.local/share/virtualenvs/investigator-bEh7fL8p/lib/python3.6/site-packages/thoth/messaging/__init__.py:20: in <module>
    from .message_base import MessageBase
../../../.local/share/virtualenvs/investigator-bEh7fL8p/lib/python3.6/site-packages/thoth/messaging/message_base.py:41: in <module>
    class MessageBase:
../../../.local/share/virtualenvs/investigator-bEh7fL8p/lib/python3.6/site-packages/thoth/messaging/message_base.py:45: in MessageBase
    ssl_context = ssl.create_default_context(purpose=ssl.Purpose.SERVER_AUTH, cafile=KAFKA_CAFILE)
/usr/lib64/python3.6/ssl.py:468: in create_default_context
    context.load_verify_locations(cafile, capath, cadata)
E   FileNotFoundError: [Errno 2] No such file or directory
======================================================================================================================= short test summary info =======================================================================================================================
ERROR tests/test_investigate_unresolved_package.py - FileNotFoundError: [Errno 2] No such file or directory
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
========================================================================================================================== 1 error in 0.55s ===========================================================================================================================
```

This PR restructure the base class to not request the cert by default unless the class is initialized.

In this way we can import only one class:

```
from thoth.messaging import UnresolvedPackageMessage
app = UnresolvedPackageMessage().app
unresolved_package_message_topic = app.topic
```

### CODE REVIEW

File: thoth/messaging/message_base.py [hunk 1]
```diff
@@ -25,47 +25,51 @@
...
-            broker=KAFKA_BOOTSTRAP_SERVERS,
-            value_serializer="json",
-        )
-
     def __init__(
         self,
-        topic_name: str = MESSAGE_BASE_TOPIC,
-        value_type=None,
+        topic_name: str = "thoth.base-topic",
+        value_type: bool = None,
```
Comment (KPostOffice) [inline thoth/messaging/message_base.py]: Minor comment, the type here should be Optional[Type]

File: thoth/messaging/message_base.py [hunk 2]
```diff
@@ -25,47 +25,51 @@
...
         num_partitions: int = 1,
         replication_factor: int = 1,
+        client_id: str = "thoth-messaging",
+        ssl_auth: Optional[int] = 1,
+        bootstrap_server: str = "localhost:9092",
+        topic_retention_time_second: str = 60 * 60 * 24 * 45,
```
Comment (KPostOffice) [inline thoth/messaging/message_base.py]: int

**Comment by pacospace** [comment]
@harshad16 pytest failing, but I cannot reproduce it locally

**Comment by pacospace** [comment]
@KPostOffice why do we have thoth-common in this component?

**Pull request merged**